### PR TITLE
Bulk temp function

### PR DESCRIPTION
**Pull request by FranklinTrevor**
This pull request changes temp bulk and temp bulk hotf into functions so they can be imported to other files for testing.

### CODE REVIEW

File: TACOCAT.py [hunk 1]
```diff
@@ -70,18 +68,28 @@ def test(x,y):
 #Core Temperature Calculations
 #Call axial bulk temperature distribution calculation
 
-FluxPro = np.zeros(Geometry.steps)
-FluxPro[:] = np.cos((np.pi/Hc)*Geometry.z[:])
-
-Tbulk = np.zeros(Geometry.steps) #Initialize Bulk Temperature of Coolant - C
-Tbulk[0] = Tbulkin
+# Bulk Temperature of Coolant in Average Channel - C
+def BulkTemp(FluxPro):
```
Comment (lcarasik) [inline TACOCAT.py]: Add the other inputs to the function bulktemp(FluxPro, geometry, Uinlet, coolant type, etc)

File: TACOCAT.py [hunk 2]
```diff
@@ -70,18 +68,28 @@ def test(x,y):
...
-FluxPro[:] = np.cos((np.pi/Hc)*Geometry.z[:])
-
-Tbulk = np.zeros(Geometry.steps) #Initialize Bulk Temperature of Coolant - C
-Tbulk[0] = Tbulkin
+# Bulk Temperature of Coolant in Average Channel - C
+def BulkTemp(FluxPro):
+	Tbulk = np.zeros(Geometry.steps) #Initialize Bulk Temperature of Coolant - C
+	Tbulk[0] = Tbulkin
+	for i in range(0,Geometry.steps):
+		Tbulk[i] = Tbulkin + (np.trapz(FluxPro[0:i+1],Geometry.z[0:i+1])*Geometry.NFuel*qlin)/(Coolant[Coolant_Type]["Cp"]*Uinlet*Coolant[Coolant_Type]["rho"]*Core_Geometry[Geometry_Type]["CoolantFlowArea"]) #Bulk Temperature of Coolant - C
```
Comment (lcarasik) [inline TACOCAT.py]: Add the other inputs to the function bulktemp(FluxPro, geometry, Uinlet, coolant type, etc)

File: TACOCAT.py [hunk 3]
```diff
@@ -70,18 +68,28 @@ def test(x,y):
...
+	return Tbulk
+
+# Bulk Temperature of Coolant in Hottest Channel - C
+def HotFBulkTemp(FluxPro):
```
Comment (lcarasik) [inline TACOCAT.py]: Add the other inputs to the function bulktemp(FluxPro, geometry, Uinlet, coolant type, etc)

**Comment by lcarasik** [comment]
Is this ready to review?

**Comment by FranklinTrevor** [comment]
> 

@lcarasik it is now ready for review

**Review by lcarasik — APPROVED** [review]
This is good for merging

**Pull request merged**